src/components/manifests/ManifestUpload.tsx

```diff
@@ -60,8 +63,9 @@ const ManifestUpload: React.FunctionComponent = () => {
             getManifestValidationErrors(authData.idToken, {
                 schema: manifestType,
                 template: file
-            }).then(errs => {
+            }).then(({ errs, feedb }) => {
```

Comment:
If you're going to do object destructuring like this, either the property names need to match those on the object you are destructuring, or you can set aliases like this:
```javascript
({errors: errs, feedback: feedb})
```
but the above fails to compile (and would produce runtime errors) because neither `errs` nor `feedb` exist as properties on `IManifestUploadResponse`.